packages/react-core/src/modules/feed/context/KnockFeedProvider.tsx

```diff
@@ -26,7 +26,7 @@ const KnockFeedContext = React.createContext<
 
 export interface KnockFeedProviderProps {
   // Feed props
-  feedId: string;
+  feedId: string | undefined;
```

Comment:
Uber nit: could use `?:` to be consistent with other props that can be undefined like color mode.